_post/2020-22-05-fredbom.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
* Item 1 – Franklin D. Roosevelt
“Human kindness has never weakened the stamina or softened the fibre of free people. A nation does not have to be cruel to be tough.”